tests/__tests__/secondBodyTest.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import SecondBody from '../../client/src/components/secondBody.jsx';
+
+test('should have initial button prior to first click', () => {
+  var component = shallow(<SecondBody moresummary={'Sample Summary'} />);
+  expect(component.find('#Hide').text()).toBe('Read more about the space');
+});
+
+test('should show second button after first click', () => {
+  var component = mount(<SecondBody moresummary={'Sample Summary'} />);
+  component.find('#Hide').simulate('click');
+  expect(component.find('#Hide').text()).toBe('Hide');
+});
+
+test('should show summary data only after initial click', () => {
+  var component = mount(<SecondBody moresummary={'Sample Summary'} />);
+  expect(component.find('#summary').exists()).toBe(false);
+  component.find('#Hide').simulate('click');
```

Comment:
Your tests do pass, but I suggest you use component.update() to allow your component to re-render after simulating an event or calling a prop function, because it doesn't happen automatically during tests. This may be more important in more complex tests.